packages/dapp/components/layout/Layout.tsx

```diff
@@ -23,13 +23,25 @@ function ErrorHandler({ error }: { error: Error }) {
 }
 
 export default function Layout({ children }: LayoutProps) {
+  const firstUpdate = useRef(false);
+  const updateFinish = useRef(false);
+
   useEffect(() => {
-    const { ethereum } = window;
-    if (ethereum) {
-      ethereum.on("accountsChanged", window.location.reload);
-      ethereum.on("chainChanged", window.location.reload);
+    //@note Fix: (Illegal Invocation)
```

Comment:
Can you elaborate?

Reply:
It fixes the Illegal Invocation, likely causing by duplicate event listeners @pavlovcik